core/src/test/scala/com/avast/clients/rabbitmq/LiveTest.scala

```diff
@@ -77,174 +77,174 @@ class LiveTest extends TestBase with ScalaFutures {
-      val latch = new CountDownLatch(20)
-
-      val cons = rabbitConnection.newConsumer("testing", Monitor.noOp()) { _: Delivery[Bytes] =>
-        latch.countDown()
-        Task.now(Ack)
-      }
-
-      cons.withResource { _ =>
-        rabbitConnection.newProducer[Bytes]("testing", Monitor.noOp()).withResource { sender1 =>
-          rabbitConnection.newProducer[Bytes]("testing2", Monitor.noOp()).withResource { sender2 =>
-            for (_ <- 1 to 10) {
-              sender1.send("test", Bytes.copyFromUtf8(Random.nextString(10))).await(500.millis)
-              sender2.send("test2", Bytes.copyFromUtf8(Random.nextString(10))).await(500.millis)
-            }
-
-            assertResult(true, latch.getCount)(latch.await(1000, TimeUnit.MILLISECONDS))
-            assertResult(0)(testHelper.getMessagesCount(queueName1))
-          }
-        }
-      }
-    }
-  }
-
-  test("timeouts and requeues messages") {
-    val c = createConfig()
-    import c._
-
-    RabbitMQConnection.fromConfig[Task](config, ex).withResource { rabbitConnection =>
-      val cnt = new AtomicInteger(0)
-
-      val cons = rabbitConnection.newConsumer("testing", Monitor.noOp()) { _: Delivery[Bytes] =>
-        cnt.incrementAndGet()
-
-        Task {
-          Ack
-        }.delayResult(800.millis) // timeout is set to 500 ms
-      }
-
-      cons.withResource { _ =>
-        rabbitConnection.newProducer[Bytes]("testing", Monitor.noOp()).withResource { sender =>
-          for (_ <- 1 to 10) {
-            sender.send("test", Bytes.copyFromUtf8(Random.nextString(10))).await
-          }
-
-          eventually(timeout(Span(3, Seconds)), interval(Span(0.25, Seconds))) {
-            assert(cnt.get() >= 40)
-            assert(testHelper.getMessagesCount(queueName1) <= 20)
-          }
-        }
-      }
-    }
-  }
-
-  test("timeouts and requeues messages, blocking the thread") {
-    val c = createConfig()
-    import c._
-
-    implicit val cs: ContextShift[IO] = IO.contextShift(ExecutionContext.global)
-    implicit val timer: Timer[IO] = IO.timer(ExecutionContext.global)
-
-    RabbitMQConnection.fromConfig[IO](config, ex).withResource { rabbitConnection =>
-      val cnt = new AtomicInteger(0)
-
-      val cons = rabbitConnection.newConsumer("testing", Monitor.noOp()) { _: Delivery[Bytes] =>
-        cnt.incrementAndGet()
-        Thread.sleep(800) // timeout is set to 500 ms
-        IO.pure(Ack)
-      }
-
-      cons.withResource { _ =>
-        rabbitConnection.newProducer[Bytes]("testing", Monitor.noOp()).withResource { sender =>
-          for (_ <- 1 to 10) {
-            sender.send("test", Bytes.copyFromUtf8(Random.nextString(10))).unsafeRunSync()
-          }
-
-          eventually(timeout(Span(5, Seconds)), interval(Span(0.25, Seconds))) {
-            assert(cnt.get() >= 40)
-            assert(testHelper.getMessagesCount(queueName1) <= 20)
-          }
-        }
-      }
-    }
-  }
+//  test("basic") {
```

Comment:
I don't like commented code. Isn't it better to remove it completely?

Reply:
The code shouldn't be commented...